### PR TITLE
implement id

### PR DESCRIPTION
**Pull request by schuer**
will be required for #37

screenshot 2017-11-08 02 57 11

### CODE REVIEW

**Comment by schuer** [comment]
Feature will be completed in JS later on. This PR is just the _backend_ part ;)

File: index.html [hunk 1]
```diff
@@ -89,6 +89,11 @@ <h1 class="popover__title">
     var directory = [
         {% for entry in site.directory %}
 
+        <!-- id -->
+        {% assign parts = entry.id | split: "/" %}
```
Comment (staabm) [inline index.html]: hmm so, this "id" will be part of the data.yml files?

who needs to take care that those IDs are unique?
who will assign IDs to persons?

Comment (schuer) [reply index.html]: They’re build from the path. It’s what users name their folders.

Comment (staabm) [reply index.html]: awesome!

File: index.html [hunk 2]
```diff
@@ -89,6 +89,11 @@ <h1 class="popover__title">
     var directory = [
         {% for entry in site.directory %}
 
+        <!-- id -->
```
Comment (staabm) [inline index.html]: maybe add a bit more description here (as we are all new to the jekyll stuff)

Comment (schuer) [reply index.html]: Yeah, you’re right. Also this Liquid templating stuff can be a little tricky sometimes.

**Comment by schuer** [comment]
I will add more and merge afterwards.

**Pull request merged**